docs/InspectingContracts.md

```diff
@@ -0,0 +1,73 @@
+# Inspecting Terra Contracts
+
+Contract schemas for Terra can be found in the [contracts](/contracts) folder. Each contract in this directory has a subdirectory called **schema**. In these folders, you can learn how to query, execute, and instantiate each contract. Query functions are located in the **query_msg.json** files.
+
+The following is the JSON representation of a possible query to the OCR2 contract pulled from [/contracts/ocr2/schema/query_msg.json](/contracts/ocr2/schema/query_msg.json).
+
+```json
+{
+    "type": "object",
+    "required": [
+        "owed_payment"
+    ],
+    "properties": {
+        "owed_payment": {
+            "type": "object",
+            "required": [
+                "transmitter"
+            ],
+            "properties": {
+                "transmitter": {
+                    "type": "string"
+                }
+            }
+        }
+    },
+    "additionalProperties": false
+}
+```
+
+This representation indicates that the query is an object that requires another object called "owed_payment" which requires a string called "transmitter". The JSON query would be of the following format.
+
+```json
+{
+    "owed_payment": {
+        "transmitter": "TRANSMITTER_ADDRESS"
+    }
+}
+```
+
+Deployed contracts for Terra can be found in the [Reference Data Directory](https://github.com/smartcontractkit/reference-data-directory). This directory includes deployments of proxies and OCR2 contracts for both [mainnet](https://github.com/smartcontractkit/reference-data-directory/blob/master/directory-terra-mainnet.json) and [testnet](https://github.com/smartcontractkit/reference-data-directory/blob/master/directory-terra-testnet-bombay.json).
```

Comment:
Please remove this section. RDD is our internal resource and we should not reference it here.